src/app/app.module.ts

```diff
@@ -4,15 +4,18 @@ import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 
 import { AppComponent } from './app.component';
+import { SignupFormComponent} from './signupform/signupform.component';
 
 @NgModule({
   declarations: [
     AppComponent,
-    HomeComponent
+    HomeComponent,
+    SignupFormComponent,
   ],
   imports: [
     BrowserModule,
-    AppRoutingModule
+    AppRoutingModule,
+    SignupFormComponent
```

Comment:
imports are where you declare modules. Modules are usually a service of something that comes from an outside source. With a component, you only need to declare it, as you did above.